config/samples/ccruntime.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: confidentialcontainers.org/v1beta1
 kind: CcRuntime
 metadata:
   name: ccruntime-sample
+  namespace: confidential-containers-system
```

Comment:
is there a need to add this back?

Reply:
I'll double check and update. Thanks for the reviews!